cfme/infrastructure/deployment_roles.py

```diff
@@ -266,9 +311,10 @@ def delete(self, cancel=False):
             assert view.is_displayed
             view.flash.assert_success_message("The selected Clusters / "
                                               "Deployment Roles was deleted")
+            view.browser.refresh()
```

Comment:
what is it for ? 
btw, I don't see a check that there is a flash message if deployment role removal has been canceled

Reply:
@izapolsk It does not display any flash message as we cancelled removal. Refresh is completely optional, it will help to reload the page so we can see its deleted.